### PR TITLE
Adjust certificate "mini cert" to be a placeholder as opposed to containing actual certificate content

### LINKED ISSUE #1950: [6.0] Pop Up Mini Certificate Oversized
### Reproduction Steps

+ create a certificate
+ complete the trigger or assign the certificate to a user
+ view the pop up for the certificate awarded on the front end

### Expected Behavior

+ You can see the pop up and click the option to view the full certificate
image

### Actual Behavior

+ In Divi the mini certificate takes up the whole screen and you can not click the view full certificate
+ It also hides any other pop ups that come at the same time
image

### Error Messages / Logs

+ Include any relevant error messages or log files
```
<!-- Paste error logs / backtraces below this line -->

```

### System and Environment Information

<details>
<summary>System Report</summary>

```
Wordpress
-------------------------------------------

Home Url: https://natalie-test-site.myliftersite.com
Site Url: https://natalie-test-site.myliftersite.com
Login Url: https://natalie-test-site.myliftersite.com/wp-login.php
Version: 5.8.3
Debug Mode: No
Debug Log: No
Debug Display: Yes
Locale: en_US
Multisite: No
Page For Posts: Not Set
Page On Front: Not Set
Permalink Structure: /%postname%/
Show On Front: posts
Wp Cron: Yes


Settings
-------------------------------------------

Version: 6.0.0-alpha.3
Db Version: 5.7.0
Course Catalog: Course Catalog (#35) [https://natalie-test-site.myliftersite.com/courses/]
Membership Catalog: Membership Catalog (#36) [https://natalie-test-site.myliftersite.com/memberships/]
Student Dashboard: Dashboard (#38) [https://natalie-test-site.myliftersite.com/dashboard/]
Checkout Page: Checkout (#715) [https://natalie-test-site.myliftersite.com/checkout/]
Course Catalog Per Page: 9
Course Catalog Sorting: menu_order,ASC
Membership Catalog Per Page: 9
Membership Catalog Sorting: menu_order
Site Membership: Not Set
Courses Endpoint: my-courses
Edit Endpoint: edit-account
Lost Password Endpoint: lost-password
Vouchers Endpoint: redeem-voucher
Autogenerate Username: no
Password Strength Meter: no
Minimum Password Strength: 
Terms Required: yes
Terms Page: Activate (#443) [https://natalie-test-site.myliftersite.com/activate/]
Checkout Names: 
Checkout Address: 
Checkout Phone: 
Checkout Email Confirmation: no
Open Registration: yes
Registration Names: 
Registration Address: 
Registration Phone: 
Registration Voucher: 
Registration Email Confirmation: no
Account Names: 
Account Address: 
Account Phone: 
Account Email Confirmation: no
Confirmation Endpoint: confirm-payment
Force Ssl Checkout: no
Country: US
Currency: USD
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Decimals: 2
Trim Zero Decimals: no
Recurring Payments: yes
Email From Address: team+sandbox@lifterlms.com
Email From Name: A LifterLMS Sandbox
Email Footer Text: 
Email Header Image: 
Cert Bg Width: 1123
Cert Bg Height: 794
Cert Legacy Compat: no


Constants
-------------------------------------------

LLMS_REMOVE_ALL_DATA: undefined
LLMS_REST_DISABLE: undefined
LLMS_SITE_FEATURE_RECURRING_PAYMENTS: undefined
LLMS_SITE_IS_CLONE: undefined


Gateways
-------------------------------------------

Authorize.Net: Enabled
Authorize.Net Test Mode: Enabled
Authorize.Net Logging: no
Authorize.Net Order: 1
PayPal: Enabled
PayPal Test Mode: Enabled
PayPal Logging: no
PayPal Order: 1
Stripe: Enabled
Stripe Test Mode: Enabled
Stripe Logging: no
Stripe Order: 1
Manual: Enabled
Manual Logging: no
Manual Order: 1


Server
-------------------------------------------

Mysql Version: 5.7.36
Php Curl: Yes
Php Default Timezone: UTC
Php Fsockopen: Yes
Php Max Input Vars: 5000
Php Max Upload Size: 512 MB
Php Memory Limit: 256M
Php Post Max Size: 1024M
Php Soap: Yes
Php Suhosin: No
Php Time Limt: 30
Php Version: 7.3.33
Software: Apache/2.4.52 (Unix) OpenSSL/1.1.1
Wp Memory Limit: 256M


Browser
-------------------------------------------

HTTP USER AGENT: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.99 Safari/537.36


Theme
-------------------------------------------

Name: Divi
Version: 4.14.7
Themeuri: http://www.elegantthemes.com/gallery/divi/
Authoruri: http://www.elegantthemes.com
Template: 
Child Theme: No
Llms Support: No


Plugins
-------------------------------------------

LifterLMS: 6.0.0-alpha.3
LifterLMS Advanced Quizzes: 1.1.1
LifterLMS Advanced Videos: 1.0.0-beta.17
LifterLMS Authorize.Net Payment Gateway: 1.1.0
LifterLMS ConvertKit: 2.1.2
LifterLMS Custom Fields: 2.0.2
LifterLMS Formidable Forms: 1.0.4
LifterLMS Gravity Forms: 2.1.2
LifterLMS Groups: 1.0.0-beta.17
LifterLMS MailChimp Add-on: 3.2.0
LifterLMS Ninja Forms: 1.1.1
LifterLMS PayPal Gateway: 1.3.0
LifterLMS PDFs: 1.0.0
LifterLMS Private Areas: 1.1.5
LifterLMS Social Learning: 1.4.0
LifterLMS Stripe Payment Gateway: 5.4.0
LifterLMS WooCommerce: 2.2.1
LifterLMS WPForms: 1.1.1
User Switching: 1.5.8
WooCommerce: 6.1.1


Integrations
-------------------------------------------

BbPress: No
BuddyPress: No
LifterLMS ConvertKit: Yes
LifterLMS Groups: Yes
LifterLMS Formidable Forms: No
LifterLMS Gravity Forms: No
LifterLMS Ninja Forms: No
WooCommerce: Yes
LifterLMS WPForms: No
LifterLMS MailChimp: Yes
LifterLMS Private Areas: Yes
LifterLMS Social Learning: Yes
Videos: Vimeo: No
Videos: Wistia: No
Videos: YouTube: Yes


Template Overrides
-------------------------------------------





```

</details>

This issue has be recreated:
+ [ ] Locally
+ [x] On a staging site
+ [ ] On a production website
+ [ ] With only LifterLMS and a default theme

### Browser, Device, and Operating System Information

+ Browser name and version
+ Operating System name and version
+ Device name and version (if applicable)

### PR DESCRIPTION
**Pull request by thomasplevy**
## Description

Closes #1950

## How has this been tested?

+ Manually

## Screenshots 

https://user-images.githubusercontent.com/1290739/152595848-b54fc890-87a8-4bad-91ff-8156e4e63bd8.mp4

### CODE REVIEW

File: assets/scss/frontend/_llms-notifications.scss [hunk 1]
```diff
@@ -67,18 +67,45 @@
...
+						}
+						.llms-mini-cert--mock-dot {
+							background: #b0b0b0;
+							border-radius: 50%;
+							display: inline-block;
+							content: '';
+							height: 24px;
+							width: 24px;
+						}
+						p { margin-bottom: 0; }
```
Comment (eri-trabiccolo) [inline assets/scss/frontend/_llms-notifications.scss]:
```suggestion
					.llms-mini-cert--mock-line {
						border-radius: 2px;
						height: 8px;
						background: #b0b0b0;
						background-image: linear-gradient( to right, #b0b0b0 0%, #a0a0a0 20%, #b0b0b0 40%, #b0b0b0 100% );
						background-repeat: no-repeat;
						margin: 4px auto;
					}
					.llms-mini-cert--mock-dot {
						background: #b0b0b0;
						border-radius: 50%;
						display: inline-block;
						content: '';
						height: 24px;
						width: 24px;
					}
					p { margin-bottom: 0; }
```

File: assets/scss/frontend/_llms-notifications.scss [hunk 2]
```diff
@@ -67,18 +67,45 @@
...
-					padding: 24px 8px;
+					border: 1px solid #d0d0d0;
+					box-shadow: inset 0 0 0 3px #fefefe, inset 0 0 0 4px #dedede;
+					padding: 16px 16px 24px;
+					margin-top: 12px;
 					.llms-mini-cert-title {
 						font-size: 16px;
 						font-weight: 700;
-						margin-bottom: 8px;
+						margin-bottom: 12px;
```
Comment (eri-trabiccolo) [inline assets/scss/frontend/_llms-notifications.scss]:
```suggestion
						margin: 12px auto;
```
Let's set the margin top too. I was using 2020 theme and it had too much space above by default.

**Review by eri-trabiccolo — CHANGES_REQUESTED** [review]
looks good overall!

**Comment by thomasplevy** [comment]
@chrisbadgett @willmiddleton-lifterlms @nrherron92 can I get your thoughts on this before merging?

**Comment by nrherron92** [comment]
@thomasplevy I like it! Without the sweeping effect I think that takes away the idea that it's loading so that should help on that front (even though I thought that looked cool too!). Regardless, I think it really nice as a mini preview!

**Comment by thomasplevy** [comment]
Bump @willmiddleton-lifterlms @chrisbadgett

**Comment by thomasplevy** [comment]
@chrisbadgett @willmiddleton-lifterlms last bump. I'm going to merge this afternoon for release with beta.1

**Comment by chrisbadgett** [comment]
Looks solid.

**Pull request merged**